tests/unit/Utopia/Nested.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Tests\Unit\Utopia;
+
+use Appwrite\Utopia\Response\Model;
+
+class Nested extends Model
```

Comment:
What exactly is this class?

Reply:
I'm not sure where this change has come from 🤔 Will investigate

Reply:
There are coming from `fix-unit-tests` which is already merged in `0.16.x`, shall I try remove? Or sync `0.16.x`?

Reply:
Sync with 0.16x